README.md

```diff
@@ -47,9 +47,23 @@ For the quickstart and Docker installation modes, [Docker](https://docker.com) n
 Please refer to the official documentation and make sure your user is added to the `docker` user group if you cannot
 access root permissions to run containers.
 
-### Backups
+### Security
+
+For validators it is crucial to safely back up their private key. If they lose their key, they will not be able to
+
+- create any blocks or earn block rewards or
+- withdraw their stake on the main chain once it is unlocked.
+
+Furthermore, it is advisable to configure validating nodes in such a way that mining rewards are not sent to the
+validator account (the default), but to a separate one. The corresponding private key can be kept in a cold wallet and
+even on a different machine, protecting it from attacks on the validator node itself.
```

Comment:
I think this would be too strong. Using a cold wallet on the same machine is already a big improvement. What exactly do you not understand about the sentence?

Reply:
Yeah, I guess it is totally fine. Nobody will care. We have the link to the documentation if someone does not understand this.